python/GangaLHCb/test/GPI/InterfaceLookFeel.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+
+class TestInterfaceLookFeel(GangaUnitTest):
+
+    def testInterfaceLookFeel(self):
+
+        from Ganga.GPI import Job, LSF, Executable, DaVinci
+
+        j1 = Job(name='my',application='DaVinci')
+        j2 = Job(application = DaVinci())
+
+        j1.backend = LSF()
+        j1.backend.queue = '8nm'
+        j2.backend = j1.backend # deepcopy
+        j2.backend.queue = '16nh' # shortcut
+        bk2 = j2.backend # reference
+
+        assert(j2.backend.queue == '16nh')
```

Comment:
`assert` should not use brackets around its arguments. It's error prone if a message is added in the future.